src/mode/modeHandler.ts

```diff
@@ -145,6 +146,7 @@ export class VimState {
    * by us or by a mouse action.
    */
   public whatILastSetTheSelectionTo: vscode.Selection;
+<<<<<<< b9a0b15742532ecc54e66ee04c417d82e70d29dc
```

Comment:
I start to doubt myself as it happens several times. I remember correctly yesterday that I did remove these lines by `dd` and save them then use `git add . & git rebase --continue`. After that, I even search `====` globally as I made this kind of mistakes before, but damn, it still happens.

I'll try to track this down next time I do rebasing. The only potential catch came to my mind now is I always reload the window as jsdiff is hanging, I don't if this leads to the unsuccessful saving.